bolt-design-system/bolt
# Video | FE | Deprecate isBackgroundVideo prop

## Jira

http://vjira2:8080/browse/BDS-1370

## Summary

Deprecate `isBackgroundVideo` prop.

## Details

Deprecate `isBackgroundVideo` prop in Twig and JS, and remove from docs. It will be removed completely in v3.0.

## How to test

- Review code changes
- Check-out locally and verify:
    - Site builds
    - Band with Background docs page no longer have background video example
    - Video with inline script and external controls as background page is gone
